components/SearchBar/index.tsx

```diff
@@ -101,14 +102,29 @@ function Component() {
 
   useEffect(() => {
     const path = window.location.pathname.replace("/", "").toLowerCase();
+    
     if (path.split('/')[1] === 'xwallet' && !is_controller) {
       Router.push(`/${path.split('/')[0]}`)
     } else if (path.includes('.did') && path.includes('/')) {
       Router.push(`/${path.split('/')[0].split('.')[0]}/${path.split('/')[1]}`)
       getResults();
+    } else if (path.includes('.tyron')) {
+      window.open(
+        SMART_CONTRACTS_URLS[
+        'xwallet'
```

Comment:
@DevPH-Wistkey xwallet.tyron is only one option for the smart contracts of the protocol ([see S4](https://www.notion.so/ssiprotocol/App-sprint4-0ea1615ca1e74c849a2a516dc1d19513#4bc82af42eff423380228deda262c376) - thanks!